.readthedocs.yml

```diff
@@ -1,7 +1,21 @@
-requirements_file: docs/requirements.txt
+version: 2
+
 build:
-  image: testing
+  os: ubuntu-22.04
+  tools:
+    python: "3.10"
+    nodejs: "19"
+  jobs:
+    post_install:
+      - npm install -g @mermaid-js/mermaid-cli
+
+sphinx:
+  configuration: docs/source/conf.py
+
 python:
-  version: 3.9
-  setup_py_install: false
-  pip_install: true
+  install:
+    - requirements: docs/requirements.txt
+    - method: pip
+      path: .
+      extra_requirements:
+        - dev
```

Comment:
Is that needed? I don't think we have any extras defined there, and the indico package itself shouldn't be installed (I think)

Reply:
ah hm, i thought we had skipped this, but maybe that was in the past when we still had this ZODB crap which failed to install in the RTD env. so nevermind, I think `[dev]` actually makes sense in that case.